### ISSUE TITLE
Java Open JDK Platform Binary after End of Game in Task

### ISSUE THREAD

**Issue by Commodusaurel** [b3dgs/lionheart-remake#358]
After the end of the game, the "OpenJDK Platform Binary" is located in the task manager.

With every new game start, another task is added, which swallows up to 400MB of RAM.

Why isn't this cleared at the end of the game?

## Specifications
  - Lionheart Remake Version: V1.3.0
  - OS:  WIN 11

**Comment by DjThunder** [comment]
I can't reproduce it. Can you start with a fresh boot, just launch the game, quit from menu and check.
Then same after started a stage.

Thanks !

**Comment by Commodusaurel** [comment]
Sorry, I can't reproduce it anymore either. Now the "OpenJDK Platform Binary" in the task manager is closed after the end of the game or configuration. I don't know why this error occurred. This ticket can probably be closed.

**Comment by Commodusaurel** [comment]
10 minutes after the end of the game I still had "OpenJDK Platform Binary" in the task manager, which occupied 265MB. I don't know why this task doesn't always close. So please don't close it

**Comment by DjThunder** [comment]
Something may be running in background (such as a music task, even without music, or another process; but I can't find what can stay alive).

If you experience it again, can you download: https://visualvm.github.io/download.html (zip)
Simply run it, double click on AppLionheart (list on left) when problem occurs, go to "threads" tab, and send me a screen, or tell me what is in Green, Yellow or Orange/Brown.

**Comment by Commodusaurel** [comment]
I installed the Java Development Kit and the recommended VisualVM216. Then I started Lionheart1.3.0 and clicked Lionheart in VisualVM216. I found only one thread dump.

I can only say that I usually have to manually remove the "OpenJDK Platform Binary" from the task manager, since Java does not close itself after the game ends.
Path: /data/jre_win32-x86_64/bin/javaw.exe

[Lionheart_Threaddump.txt](https://github.com/b3dgs/lionheart-remake/files/11562196/Lionheart_Threaddump.txt)

**Comment by DjThunder** [comment]
Thx a lot.
Can you check .lionengine file in your %TMP% folder ?

**Comment by Commodusaurel** [comment]
Yes, I only had Java installed, not the Java Development Kit, and tried to start VisualVM and Lionheart, it crashed. My mistake! That has nothing to do with the problem.

**Comment by DjThunder** [comment]
Can you override the file in data/ with this one: lionheart.b3dgs.com/files/1.3.1/patch/lionheart-pc-1.3.0.jar
And if you experience it again, use visualvm again (I will have more information on what can be guilty).

**Comment by Commodusaurel** [comment]
The problem that the TASK "OpenJDK Platform Binary" does not close automatically after the end of the game still exists with V1.3.1 (also the joypad problem with the XBOX360 controller) / Sorry....but my support is ending for now due to time constraints. Anyway thanks for everything!

Maybe I'll get back to playing more in July

**Comment by DjThunder** [comment]
Ok, if you can just send me a threadump with this version, it will be perfect. Anyway, thanks a lot for your time !

**Comment by Commodusaurel** [comment]
OK. I've played Lionheart twice with the XBox controller. Only after about 1 minute there was the control problem (jump + right direction blocked).

Lionheart is still active here:
![Lionheart_active](https://github.com/b3dgs/lionheart-remake/assets/97169656/2879082f-8153-475b-9f32-8501439c6e1e)

Here is where Lionheart ended: (A previous session of javaw.exe can still be seen in the task manager)
![Lionheart_inactive](https://github.com/b3dgs/lionheart-remake/assets/97169656/5076598b-b05f-46b0-a4c5-a649cd801570)

The javaw.exe process does not exit after the game ends:
![Processe](https://github.com/b3dgs/lionheart-remake/assets/97169656/7cbb1299-10e3-4914-a991-ae05715de982)

[Lionheart_Threaddump.txt](https://github.com/b3dgs/lionheart-remake/files/11582452/Lionheart_Threaddump.txt)

This is the original dump with the addition.txt:
[threaddump-1685203048119.tdump.txt](https://github.com/b3dgs/lionheart-remake/files/11582453/threaddump-1685203048119.tdump.txt)

**Comment by DjThunder** [comment]
Thx I found and reproduced. It was because of HQ3X filter (not closing properly).

**Comment by Commodusaurel** [comment]
Really? Great that you found it!
